stress/common/src/main/java/alluxio/stress/common/FileSystemParameters.java

```diff
@@ -46,13 +55,28 @@ public FileSystemClientType convert(String value) {
   }
 
   /**
-   * @return FileSystemClientType of this bench
    * Converts from String to FileSystemClientType instance.
+   * @return FileSystemClientType of this bench
    */
   public static class FileSystemParametersReadTypeConverter implements IStringConverter<ReadType> {
     @Override
     public ReadType convert(String value) {
       return ReadType.fromString(value);
     }
   }
+
+  /**
+   * Converts from String to a valid FileSystemWriteType String.
```

Comment:
You can also use `{@link WriteType}` so your comment will point to the real code and stay updated.